event-adapter/pkg/apiserver/apiserver.go

```diff
@@ -17,6 +14,8 @@ limitations under the License.
 package apiserver
 
 import (
+	"github.com/GoogleCloudPlatform/k8s-stackdriver/event-adapter/pkg/provider"
+	"github.com/GoogleCloudPlatform/k8s-stackdriver/event-adapter/pkg/types"
```

Comment:
Is the event API defined somewhere else, like k8s.io/metrics/pkg/apis/custom_metrics for Custom Metrics? If so, can you either import it from that location instead of including in this repo? If you prefer (and if it impacts your code), you can add a TODO to do this in the future.